src/realm/object-store/sync/impl/sync_metadata.cpp

```diff
@@ -193,6 +193,13 @@ void migrate_to_v7(std::shared_ptr<Realm> old_realm, std::shared_ptr<Realm> real
     }
 }
 
+void migrate_to_v8(Realm&, Realm& realm)
+{
+    if (auto app_metadata_table = realm.read_group().get_table("class_AppMetadata")) {
```

Comment:
Will we need to do anything to convert the `fileActionMetadata` entries to the new format without the `partition` and `identity` fields? If so, maybe add a comment somewhere in this function.

Reply:
No, removing columns no longer present in the schema is done automatically. It just requires a schema version bump since it's a destructive change.